### DISCUSSION TITLE
Improve description/review/notes fields

### DISCUSSION THREAD

**Discussion by mateusz-bak** [mateusz-bak/openreads#718] [Ideas]
**Is your feature request related to a problem? Please describe.**
I find that the 5000 word limit on notes is often too tight.
I also miss the ability to insert images directly into notes, especially photos of book pages.

**Describe the solution you'd like**
An increase to 10000+ words would be very important. 
A camera function for the direct integration of images or a separate tab for this would be great. It would also be handy to be able to open the camera directly in this case and when adding a cover image, instead of having to open the file manager first. 

**Describe alternatives you've considered**

**Additional context**

**Comment by mateusz-bak** [comment]
I think a rich text editor would work great.

**Comment by lucastucious** [comment]
Maybe even using markdown syntax for lightweight format?

**Comment by Ashkanph** [comment]
I think 10,000 WORDS would be amazing! However, the current limit of 5,000 CHARACTERS is insufficient for my reviews and notes. Sometimes, I need more than 25,000 characters for my notes alone (apart from my review).

**Comment by mateusz-bak** [comment]
Also proposed in https://github.com/mateusz-bak/openreads/discussions/849
